node/pkg/processor/observation.go

```diff
@@ -247,36 +247,10 @@ func (p *Processor) handleInboundSignedVAAWithQuorum(ctx context.Context, m *gos
 		return
 	}
 
-	// Check if VAA doesn't have any signatures
-	if len(v.Signatures) == 0 {
-		p.logger.Warn("received SignedVAAWithQuorum message with no VAA signatures",
-			zap.String("digest", hash),
-			zap.Any("message", m),
-			zap.Any("vaa", v),
-		)
-		return
-	}
-
-	// Verify VAA has enough signatures for quorum
-	quorum := CalculateQuorum(len(p.gs.Keys))
-	if len(v.Signatures) < quorum {
-		p.logger.Warn("received SignedVAAWithQuorum message without quorum",
-			zap.String("digest", hash),
-			zap.Any("message", m),
-			zap.Any("vaa", v),
-			zap.Int("wanted_sigs", quorum),
-			zap.Int("got_sigs", len(v.Signatures)),
-		)
-		return
-	}
-
-	// Verify VAA signatures to prevent a DoS attack on our local store.
-	if !v.VerifySignatures(p.gs.Keys) {
-		p.logger.Warn("received SignedVAAWithQuorum message with invalid VAA signatures",
-			zap.String("digest", hash),
-			zap.Any("message", m),
-			zap.Any("vaa", v),
-		)
+	vError := v.Verify(p.gs.Keys)
+	if vError != nil {
```

Comment:
This error variable is `vError` but we end up logging `err` on line 253.  Let's stick with the go convention of always calling the error `err`.  We can also combine the verification and error check into a single line:

```suggestion
	if err := v.Verify(p.gs.Keys); err != nil {
```

Reply:
Fixed